persistence-modules/spring-hibernate-6/src/test/resources/.gitignore

```diff
@@ -0,0 +1,13 @@
+*.class
+
+#folders#
```

Comment:
I don't think we need multiple gitignores, we already have one that is defined at the repo root. Let's remove these